sdk/formrecognizer/ai-form-recognizer/src/generated/generatedClientContext.ts

```diff
@@ -30,10 +30,10 @@ export class GeneratedClientContext extends coreClient.ServiceClient {
       options = {};
     }
     const defaults: GeneratedClientOptionalParams = {
-      requestContentType: "application/json; charset=utf-8"
+      requestContentType: "application/json; charset=utf-8",
```

Comment:
I think auto-generated files should not be formatted.

Reply:
Just a result of manually editing the file and my editor formatting on save. I've already built a release artifact against the merge of this PR and main, so I don't want to toss it and test another one if there's no harm in this.

Reply:
sounds good to me!